tests/IsbnTest.php

```diff
@@ -57,6 +57,12 @@ public function testFormatIsbn10()
         $this->assertEquals($isbn10->format('ISBN-10'), "3-464-60352-0");
     }
 
+	public function testFormatGtin14()
+	{
+		$isbn = new Isbn('9783464603529');
+		$this->assertEquals($isbn->format('GTIN-14'), '19783464603526');
+	}
```

Comment:
You can run test locally by running `composer install` (once) and then `composer test`. You'll notice that your test is passing but your changes have broken some ISBN-13 related tests.

Also CI was broken when your opened your pull request but if [you rebase your branch](https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request) against master or develop, they will be run by Github when you push.

Reply:
Got it figured out and rebased! Also figured out the logical error that made the ISBN-13 formatting go haywire. I’d got my odds and evens mixed up when moving from a 13-item array to a 14-item one, so now the array gets reversed and then the new key is used instead.